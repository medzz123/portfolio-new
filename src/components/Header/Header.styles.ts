import { styled } from '@theme/config';

export const HeaderContainer = styled('header', {
  padding: '0 $20',
  position: 'absolute',
  top: '$60',
  left: '$half',
  width: '$full',
  maxWidth: '$6xl',
  transform: 'translateX(-$sizes$half)',
});
