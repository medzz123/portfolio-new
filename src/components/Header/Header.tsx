import React, { FunctionComponent } from 'react';

import { HeaderProps } from './Header.models';
import { HeaderContainer } from './Header.styles';

const Header: FunctionComponent<HeaderProps> = () => {
  return <HeaderContainer data-testid="header">This is the header</HeaderContainer>;
};

export default Header;
