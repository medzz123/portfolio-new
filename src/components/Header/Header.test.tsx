import { render } from '@test/library';
import React from 'react';

import Header from './Header';

test('Header Test', () => {
  const { container } = render(<Header>Hello</Header>);

  expect(container).toMatchSnapshot();
});
