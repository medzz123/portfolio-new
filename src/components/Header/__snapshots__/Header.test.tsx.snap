// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Test 1`] = `
<div>
  <header
    class="sxdu0hc"
    data-testid="header"
  >
    This is the header
  </header>
</div>
`;
